recipe/bld.bat

```diff
@@ -16,6 +16,12 @@ IF "%CONDA_PY%"=="35" (
     SET ENABLE_CXX11=NO
 )
 
+IF "%CONDA_PY%"=="36" (
```

Comment:
I think this is redundant now, as above it's already checking for 35. I'll remove it.